start/src/main/java/org/apache/accumulo/start/spi/KeywordExecutable.java

```diff
@@ -28,7 +28,7 @@
  * All implementing classes who have an entry in
  * META-INF/services/{@link org.apache.accumulo.start.spi.KeywordExecutable} on the classpath will
  * be constructed by the {@link ServiceLoader}, so they should be lightweight and quickly
- * constructible with a mandatory no-argument constructor. Because of this, implementing classes
```

Comment:
Looks like its debatable but "constructible" might be the correct US-en spelling.

Reply:
I will go ahead and revert this change.